src/controllers/requestController.js

```diff
@@ -0,0 +1,23 @@
+import RequestService from '../services/requestService';
+import Response from '../utils/response';
+/**
+ * Trip requests
+ */
+export default class RequestController {
+/**
+ * @param {object} body request body
+ * @param {object} res response body
+ * @returns {object} res
+ */
+  static async oneWay({ body }, res) {
+    try {
+      const data = await RequestService.oneway(body);
+
+      Response.success(res, data, 201);
+    } catch ({ message: error }) {
+      if (error === 'You\'ve already booked this trip') return Response.badRequest(res, error, 409);
+
+      Response.badRequest(res, error);
```

Comment:
you can use object destructuring here e.g const { success, badRequest} = Response
so you can have success(res, data, 201)

Reply:
okay, noted